content/blog/2024-08-28-owasp-modsecurity-new-versions.md

```diff
@@ -17,21 +18,21 @@ It's been a long time since the last releases, especially in case of v2.
 #### Major changes in v2:
 
 * added a CI workflow
-* changed error log format - plase take a review to [PR](https://github.com/owasp-modsecurity/ModSecurity/pull/3192)
-* added a new MULTIPART HEADER check - please take a review to [PR](https://github.com/owasp-modsecurity/ModSecurity/pull/3226)
-* fixed many possible memory leaks and other possible memory handling problems
+* changed error log format - [PR](https://github.com/owasp-modsecurity/ModSecurity/pull/3192)
+* added a new MULTIPART HEADER check - [PR](https://github.com/owasp-modsecurity/ModSecurity/pull/3226)
+* fixed many potential memory leaks and other potential memory handling problems
 
 
 #### Major changes in v3:
 
-* added Windows port - thanks to [@eduar-hte](https://github.com/owasp-modsecurity/ModSecurity/pull/3132)
-* improved CI workflow - also @eduar-hte
-* removed unnecessary string copies, improved engine speed - @eduar-hte
-* fixed a bug in `@pm` operator - [@eduar-hte](https://github.com/owasp-modsecurity/ModSecurity/pull/3233) - please take a look, the operator's behavior has fixed!
```

Comment:
I agree we don't need to notice users here, but I'm sure we have to notice them some way about the new behavior. Or shouldn't we?

Reply:
That's why I had added the line at the top.